tools/build.sh

```diff
@@ -360,13 +360,31 @@ awk '
 # make redirect file available at /request/ too
 cp dist/index.html dist/request
 
+# Build RSA iframe
+output "🔑  Building RSA Iframe"
+cp src/lib/rsa/sandboxed/forge.min.js dist/lib/rsa/sandboxed/forge.min.HASH.js
+# Integirty check that forge.min.js is from node-forge 1.3.1
+RSA_IFRAME_FORGE_HASH="3Gf9EyQnrZbJZmyESzlWVBPEDdsfLQY8U1Evv204ff0="
```

Comment:
Why hard-code the hash here but not verify it? That just means it will fail at runtime if we update the script, instead of at build time. 

Either:
1. Calculate the hash from the script that is copied to the build folder
2. Verify this hash against the file it is supposed to represent